examples/operator/101_initial_cluster.yaml

```diff
@@ -0,0 +1,187 @@
+# The following example is minimalist. The security policies
+# and resource specifications are not meant to be used in production.
+# Please refer to the operator documentation for recommendations on
+# production settings.
+apiVersion: planetscale.com/v2
+kind: VitessCluster
+metadata:
+  name: example
+spec:
+  images:
+    vtctld: us.gcr.io/planetscale-vitess/lite:2020-04-24.228e6fe
```

Comment:
Any particular reason to pin the version? Local examples run against latest code/build.